frontend/ui/screensaver.lua

```diff
@@ -410,8 +423,30 @@ function Screensaver:show(event, fallback_message)
                 }
             }
         end
+
         -- No overlay needed as we just displayed the message
         overlay_message = nil
+
+        -- check if message_widget should be overlaid on another widget
+        if message_widget ~= nil then
+            if widget ~= nil
+            then
```

Comment:
On a single line please :) And even just `if widget then -- we have a screensaver widget`